Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/VirtualReality.java

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package org.dolphinemu.dolphinemu.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+
+import org.dolphinemu.dolphinemu.BuildConfig;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Locale;
+
+public class VirtualReality
+{
+  private static final String VR_PACKAGE = "org.dolphinemu.dolphinemu.vr";
+
+  public static boolean isActive()
+  {
+    return BuildConfig.BUILD_TYPE.equals("vr");
+  }
+
+  public static boolean isInstalled(Context context)
+  {
+    PackageManager pm = context.getPackageManager();
+    for (ApplicationInfo app : pm.getInstalledApplications(PackageManager.GET_META_DATA))
+    {
+      if (app.packageName.equals(VR_PACKAGE))
```

Comment:
This doesn't handle debug builds, which have `.debug` appended to the package name. (Unless you're planning on not doing that for the VR builds?)

Reply:
This method detects if the VR APK is installed.

The problem with OpenXR is that it cannot render Android layouts. For now I use usual Android app (doesn't matter if debug or release) as a launcher. If the "launcher" detects there is VR build installed (using the method above) then it starts the games using a VR build.

Reply:
Yes, but what I mean is that it doesn't handle the VR APK having `.debug` appended to its package name. Is that a mistake, or are you planning to not have the `.debug` suffix for VR APKs?